Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
    import modulo_desconocido
ModuleNotFoundError: No module named 'modulo_desconocido'